SwrveSDK/src/main/java/com/swrve/sdk/SwrveWakefulService.java

```diff
@@ -28,10 +28,10 @@ protected void onHandleIntent(Intent intent) {
             if (eventsExtras != null && eventsExtras.size() > 0) {
                 handleSendEvents(eventsExtras);
             } else {
-                SwrveLogger.e(LOG_TAG, "SwrveWakefulService: Unknown intent received.");
+                SwrveLogger.e(LOG_TAG, "SwrveWakefulService: Unknown intent received (extras: " + intent.getExtras().get(EXTRA_EVENTS) + ").");
```

Comment:
:+1: but intent.getExtras() is nullable, check it first
http://developer.android.com/reference/android/content/Intent.html#getExtras()